test/CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ function(add_fmt_executable name)
   #   https://gcc.gnu.org/bugzilla/show_bug.cgi?id=95353
   if (CMAKE_CXX_COMPILER_ID MATCHES "GNU" AND
       NOT CMAKE_CXX_COMPILER_VERSION VERSION_LESS 7.0)
+    target_compile_options(${name} PRIVATE -Wno-stringop-overflow)
     target_link_libraries(${name} -Wno-stringop-overflow)
```

Comment:
Looks like this line should be removed because `-Wno-stringop-overflow` is a compiler not linker flag.

Reply:
Linker flag is required when LTO is enabled.

Reply:
Really? Can it give a warning at link time?

Reply:
Yes.
See warning from PR #2442.

Reply:
Oh, wow. Could you add a short comment that the linker flag is needed for LTO then?

Reply:
Comment added.